libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -197,6 +206,7 @@ type family TeamFeatureStatus (a :: TeamFeatureName) :: * where
 type FeatureHasNoConfig (a :: TeamFeatureName) = (TeamFeatureStatus a ~ TeamFeatureStatusNoConfig) :: Constraint
 
 -- if you add a new constructor here, don't forget to add it to the swagger (1.2) docs in "Wire.API.Swagger"!
+-- TODO: Delete!
```

Comment:
if you want to merge this `TODO` into develop, you should make it a `FUTUREWORK`.  also, could you elaborate a little on what and why?  (`Doc.Model` is swagger 1.2, and the new end-points are documented in 2.0 now, something like that.)

Or can you just remove it in this PR?

(Same questions apply to a few comments above and below.)

Reply:
I think it should be fine to just delete them in this PR. I don't think they are used anywhere else.

Reply:
I'll remove the comment from here, but I will keep it in that other PR for classified domains and take care of that there (I am not sure at this moment if I can safely delete these).

Reply:
I got rid of these models for Swagger 1.2 and removed TODOs.